README.md

```diff
@@ -62,6 +62,7 @@ Package Name | Version | Docs
 @databases/mysql-schema-cli | [![NPM version](https://img.shields.io/npm/v/@databases/mysql-schema-cli?style=for-the-badge)](https://www.npmjs.com/package/@databases/mysql-schema-cli) | Not documented yet
 @databases/mysql-schema-introspect | [![NPM version](https://img.shields.io/npm/v/@databases/mysql-schema-introspect?style=for-the-badge)](https://www.npmjs.com/package/@databases/mysql-schema-introspect) | Not documented yet
 @databases/mysql-schema-print-types | [![NPM version](https://img.shields.io/npm/v/@databases/mysql-schema-print-types?style=for-the-badge)](https://www.npmjs.com/package/@databases/mysql-schema-print-types) | Not documented yet
+@databases/node_modules | [![NPM version](https://img.shields.io/npm/v/@databases/node_modules?style=for-the-badge)](https://www.npmjs.com/package/@databases/node_modules) | Not documented yet
```

Comment:
It looks like you must have `packages/node_modules` in your local repo, which is messing up the code-gen. Please remove this directory and the references to it.